Google.Ads.GoogleAds/examples/CampaignManagement/ValidateAd.cs

```diff
@@ -164,12 +168,18 @@ public void Run(GoogleAdsClient client, long customerId, long adGroupId)
                             }
                             count++;
                         });
+                } else {
+                    Console.WriteLine("Failure:");
+                    Console.WriteLine("The responsive search ad being added had validation errors, " +
+                        "but not the ones we expected.");
+                    Environment.Exit(1);
                 }
             }
             catch (Exception e)
             {
                 Console.WriteLine("Failure:");
                 Console.WriteLine($"Message: {e.Message}");
+                Environment.Exit(1);
```

Comment:
Please do not do Exit(1) in the code example anywhere. We do appropriate exit codes in Program.cs.

Reply:
Will the exit logic in `Program.cs` exit with a non-zero code if line 175 is removed? We manually exit with `1` in Python to indicate that non-PolicyFindingError was received. Python is the only language that manually exited, which is how we discovered that this example was flawed across all languages, so it's important that we can distinguish between a PolicyFindingError and all other API errors.

Reply:
I think the right way to do this might be to rethrow the exception in case we want `Program.cs` to handle it, so that we basically treat it as an unhandled failure.

Reply:
We just need to rethrow and do nothing. 

Main() can print the error if invoked directly. If invoked through Program.cs (which is the default entry point), then I think we need to catch the error and forward the error code for FT to work correctly.

For now, let's remove these Exit(1) calls, we need to update all code examples. We will do that in a future iteration.